### ISSUE TITLE
model.save(path, true) is substantially slower than File.open |f| { f.puts model.to_s } (Bugzilla #1274)

### ISSUE THREAD

**Issue by axelstudios** [NREL/OpenStudio#347]
On 2013-06-05 20:42:47, @macumber wrote nothing.

**Comment by macumber** [comment]
Thanks to @mbadams5 for pointing out that a sort is happening on save.  We want to sort when saving IDF files but not OSM.

https://github.com/NREL/OpenStudio/blob/develop/openstudiocore/src/utilities/idf/Workspace.cpp#L1923

**Comment by jmarrec** [comment]
Permalink for what Dan linked to for future ref: 

Original: https://github.com/NREL/OpenStudio/blob/fcd9a4317aa543cfe16a92010df2d6b92e8b0989/openstudiocore/src/utilities/idf/Workspace.cpp#L1923

current develop (circa 3.0.0): https://github.com/NREL/OpenStudio/blob/d05bc3a9ba090cf986cde56b7616b7dc445ac2f1/src/utilities/idf/Workspace.cpp#L1912

**Comment by jmarrec** [comment]
**benchcode.rb**

```ruby
m = OpenStudio::Model::exampleModel()

model_save_t = Time.now
m.save("m_save.osm", true)
model_save_t = Time.now - model_save_t
puts "model.save: #{model_save_t}"

file_save_t = Time.now
File.open('file_save.osm', 'w') { |f| f << m.to_s }
file_save_t = Time.now - file_save_t
puts "File.open: #{file_save_t}"
```

With 3.1.0:

```
(py39)julien@OS-build$ openstudio benchcode.rb 
model.save: 0.018707139
File.open: 0.014084311
```

With develop, that includes my fix for #4187 in #4188: https://github.com/NREL/OpenStudio/pull/4188

```
(py39)julien@OS-build$ $os_build_rel/Products/openstudio benchcode.rb 
model.save: 0.015108821
File.open: 0.014477103
```

So closing, performance is basically the same
